### DISCUSSION TITLE
Enhancing Logo Component: Introducing `brandLogo` Method - Feedback Needed

### DISCUSSION THREAD

**Discussion by ousid** [filamentphp/filament#8526] [Ideas]
Recently found myself in the process of potentially overriding the existing logo component in our project. While exploring the component's code, I noticed an interesting opportunity for improvement.

The current implementation checks for the existence of a `brandName()`, and If it's present, the name takes precedence. However, if the `brandName()` is missing, the default project name is displayed.

This got me thinking: What if we introduce a new method called `brandLogo()` that gives developers more control over this behavior? With `brandLogo()`, developers can decide whether they want to display a raw brand name or specify a custom logo path.

Here's how I envision the steps to implement this enhancement:

## Steps to Produce:

- Modify the existing logo.blade.php component to include a check for the presence of both `brandName` and `brandLogo`.
- If a logo path is provided (`brandLogo`), it will be displayed by default.
- If the `brandLogo` path is null, the component will fall back to displaying the `brandName`.

Now, the question is, do you find this idea interesting and valuable enough to warrant a Pull Request (PR)? I believe this enhancement could offer more flexibility and customization options to developers working with the logo component.

I would appreciate your feedback on whether this should be pursued as a `PR` and any thoughts or suggestions you may have.

cc @danharrin

**Comment by danharrin** [comment]
Yep absolutely fine with that, we do need to be careful with logo sizing though, maybe `h-9` by default. And using the `brandName()` value as the alt text for that image.
